src/utils/useLocationUpdateEffect.ts

```diff
@@ -49,6 +80,58 @@ export const leaveRoom = (user: UserInfo) => {
   });
 };
 
+export interface EnterRoomWithCounting {
+  user?: UserInfo;
+  profile?: User;
+  venue: WithId<AnyVenue>;
+  room?: AnyRoom;
+}
+
+export const openRoomWithCounting = ({
+  user,
+  profile,
+  venue,
+  room,
+}: EnterRoomWithCounting) => {
+  if (!room) {
+    // TODO: @debt this isn't going to update counting at the moment
```

Comment:
You should call `updateLocationData` with just the venue to perform the update.

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/897/commits/7362fb9322a1ab624aac307cffc111da7b9b07fe